core/templates/dev/head/pages/profile-page/profile-page.scripts.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive scripts for the Oppia profile page.
```

Comment:
One question -- I thought I've seen the naming convention for files like this be `...Requires` as well.

E.g. extensions/rich_text_components/richTextComponentsRequires.ts

I'm fine with convention we go with but it really needs to be consistent across the codebase please.

/cc @vojtechjelinek

Reply:
We have decided to go for *.scripts.ts for all. This convention can be applied to the *Requires.ts once we apply the new naming convention to all of extensions/. But for now, the extensions/ can have *Requires.ts